vendor/bat-native-ads/src/bat/ads/internal/unittest_util.cc

```diff
@@ -230,6 +230,8 @@ bool GetNextUrlEndpointResponse(
 
     // uuid exists so increment endpoint index
     url_endpoint_response_indexes_iter->second++;
+
+    url_endpoint_response_index = url_endpoint_response_indexes_iter->second;
```

Comment:
Fixes a bug in the unit test utils where MockUrlRequest did not work for chained requests. This was the first unit test to use this feature and therefore, did not affect any other tests